flake.nix

```diff
@@ -40,6 +40,28 @@
         # From https://nixos.wiki/wiki/Google_Cloud_SDK
         gdk = pkgs.google-cloud-sdk.withExtraComponents ([ pkgs.google-cloud-sdk.components.gke-gcloud-auth-plugin ]);
 
+        customProtocGenGo = pkgs.buildGoModule rec {
+          name = "protoc-gen-go";
+          owner = "protocolbuffers";
+          repo = "protobuf-go";
+          rev = "v1.30.0"; 
+          src = pkgs.fetchFromGitHub {
+            owner = "protocolbuffers";
+            repo = "protobuf-go";
+            rev = rev;
+            # This should be updated whenever rev changes!
+            # To update, set to "", run nix-shell, insert new hash
```

Comment:
Where does the new hash come from?

Reply:
Can you rather integrate this into `scripts/update-flake.sh` pls?
The less we make folks think, the more likely they are to do the right thing.

Reply:
yeah that would be better, I will do that